src/main/webapp/app/view/modal-outlet.routes.ts

```diff
@@ -11,13 +11,14 @@ import { UserRouteRoleAccessService } from 'app/core/auth/user-route-role-access
 import { CreateProjectModalComponent } from 'app/view/create-project/create-project-modal.component';
 import { ProjectAdminUpdateModalComponent } from 'app/view/project/admin/project-admin-update-modal.component';
 import { ProjectResponsibilityModalComponent } from 'app/view/project/admin/responsibilities/project-responsibility-modal.component';
-import { ProjectUserModalComponent } from 'app/view/project/admin/users/project-user-modal.component';
+import { ProjectUserInviteModalComponent } from 'app/view/project/admin/users/project-user-invite-modal.component';
 import { ProjectLocationModalComponent } from 'app/view/project/admin/locations/project-location-modal.component';
 import { ProjectSectionModalComponent } from 'app/view/project/admin/locations/sections/project-section-modal.component';
 
 import { EventUpdateModalComponent } from 'app/view/project/screenplay/event/event-update-modal.component';
 
 import { ADMIN, CONTRIBUTOR, SECRETARY } from 'app/shared/constants/role.constants';
+import { InvitationResolve } from 'app/entities/invitation/invitation.resolve';
```

Comment:
move import to other resolvers, in alphabetical order, please.

Reply:
![200](https://user-images.githubusercontent.com/25194435/104168952-1cce9a80-53ff-11eb-8ef0-c74cfd48f4c2.gif)

Reply:
can be arranged (: